dali/pipeline/operators/decoder/nvjpeg_decoder.cu

```diff
@@ -31,12 +31,12 @@ Output of the decoder is on the GPU and uses `HWC` ordering.)code")
   .AddOptionalArg("device_memory_padding",
       R"code(Padding for nvJPEG's device memory allocations.
 This parameter helps to avoid reallocation in nvJPEG whenever a bigger image
-is encountered and internal buffer needs to be reallocated to decode it. Default is 16MB.)code",
+is encountered and internal buffer needs to be reallocated to decode it.)code",
```

Comment:
How about leaving some info that the units are in bytes?

Reply:
Done